Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Revolvers/marine_revolvers.yml

```
@@ -0,0 +1,101 @@
+- type: entity
+  parent: [BaseItem, CMBaseWeaponGun, RMCBaseAttachableHolder]
+  abstract: true
+  id: RCMWeaponRevolverBase
+  components:
+  - type: AmmoCounter
+  - type: Sprite
+    state: icon
+  - type: Item
+    size: Normal
+  - type: Tag
+    tags:
+    - Sidearm
+  - type: Clothing
+    slots:
+    - suitStorage
+    - belt
+  - type: Wieldable
+  - type: Gun
+    fireRate: 1.5
+    minAngle: 10
+    maxAngle: 20
+    angleIncrease: 5
+    angleDecay: 10
+    selectedMode: SemiAuto
+    availableModes:
+    - SemiAuto
+    soundGunshot:
+      path: /Audio/_RMC14/Weapons/Guns/Gunshots/gun_44mag.ogg
+  - type: GunWieldBonus
+    minAngle: -10
+    maxAngle: -10
+    angleIncrease: -4
+    angleDecay: 0
+  - type: ContainerContainer
+    containers:
+      revolver-ammo: !type:Container
+  - type: RevolverAmmoProvider
+    whitelist:
+      tags:
+        - CMMagazineRevolver
+        - CMSpeedLoaderM44
+        - CMCartridgeRevolver44
+    proto: CMCartridgeRevolver44
+    capacity: 6
```

```suggestion
    capacity: 7
```
It does in fact hold 7 bullets in the revolver

---

this is the base revolver